<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Qt4Sozluk</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources/qt4sozluk.qrc" >:/qt4sozluk.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="1" column="0" >
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Keyword:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="keyword" />
      </item>
      <item>
       <widget class="QPushButton" name="searchButton" >
        <property name="text" >
         <string>&amp;Search</string>
        </property>
        <property name="icon" >
         <iconset resource="../resources/qt4sozluk.qrc" >:/find.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" >
     <widget class="QTextEdit" name="resultBrowser" >
      <property name="readOnly" >
       <bool>true</bool>
      </property>
      <property name="html" >
       <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>&lt;/body>&lt;/html></string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionConfigure" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_Qt4Sozluk" />
    <addaction name="actionAbout_Qt" />
   </widget>
   <widget class="QMenu" name="menuHistory" >
    <property name="title" >
     <string>Hi&amp;story</string>
    </property>
    <addaction name="actionHistoryClear" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuHistory" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionHelp" >
   <property name="text" >
    <string>&amp;Index</string>
   </property>
   <property name="shortcut" >
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout_Qt4Sozluk" >
   <property name="icon" >
    <iconset resource="../resources/qt4sozluk.qrc" >:/qt4sozluk.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;About Qt4Sozluk</string>
   </property>
  </action>
  <action name="actionAbout_Qt" >
   <property name="text" >
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="icon" >
    <iconset resource="../resources/qt4sozluk.qrc" >:/exit.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut" >
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="actionConfigure" >
   <property name="icon" >
    <iconset resource="../resources/qt4sozluk.qrc" >:/configure.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Configure Qt4Sozluk...</string>
   </property>
  </action>
  <action name="actionShow_Toolbar" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Show &amp;Toolbar</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionShow_Search_Button" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Show &amp;Search Button</string>
   </property>
  </action>
  <action name="actionBack" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>Back</string>
   </property>
  </action>
  <action name="actionForward" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>Forward</string>
   </property>
  </action>
  <action name="actionClear_history" >
   <property name="text" >
    <string>Clear</string>
   </property>
  </action>
  <action name="actionClearHistory" >
   <property name="text" >
    <string>Clear</string>
   </property>
  </action>
  <action name="actionHistoryClear" >
   <property name="text" >
    <string>&amp;Clear</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>keyword</tabstop>
  <tabstop>searchButton</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/qt4sozluk.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>keyword</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>189</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel" >
     <x>353</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
